cppcache/benchmark/CMakeLists.txt

```diff
@@ -19,12 +19,15 @@ add_executable(cpp-benchmark
   GeodeHashBM.cpp
   GeodeLoggingBM.cpp
   NoopBM.cpp
-  )
+  PdxTypeBM.cpp)
+
+#add_dependencies(integration-framework)
 
 target_link_libraries(cpp-benchmark
   PUBLIC
     apache-geode-static
     benchmark::benchmark
+    integration-framework
```

Comment:
Don't add this here. There is an cppcache/integration/benchmark already. Move the benchmark to the integration benchmark directory.

Reply:
done!